staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -524,6 +524,17 @@ type VirtualMachineInstanceGuestOSInfo struct {
 	ID string `json:"id,omitempty"`
 }
 
+// MigrationConfigSource indicates the source of migration configuration.
+//
+// +k8s:openapi-gen=true
+type MigrationConfigSource string
+
+const (
+	ClusterWideConfig     MigrationConfigSource = "ClusterWideConfig"
+	MigrationPolicyConfig MigrationConfigSource = "MigrationPolicyConfig"
+)
+
```

Comment:
where is this used?

Reply:
Yup, these are leftovers, thanks for catching that